presto-native-execution/entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo "node.id=$HOSTNAME" >> /opt/presto-server/etc/node.properties
```

Comment:
Is this line needed for native Presto?
It was deleted in standard Presto `entrypoint.sh` in the PR https://github.com/prestodb/presto/pull/22405.

Reply:
This kind of syntax (as proposed `node.id=${ENV:HOSTNAME}`) does not work in the native worker. It expects an actual value and not an "alias".

Reply:
Got it, but can we delete this line completely and use autogenerated `node.id` by default? I’m asking, because `node.id` is optional property for Java and Native workers https://github.com/prestodb/presto/pull/22815.

Reply:
ok, let me remove node.id.